Makefile

```diff
@@ -110,9 +137,6 @@ test-mutation: verify-no-changes fetch-mutation ## Run mutation tests in manual
 	echo -e "## Verifying diff ## \n##Mutations tests actually changes the code while running - this is a safeguard in order to be able to easily revert mutation tests changes (in case mutation tests have not completed properly)##"
 	./hack/test-mutation.sh
 
-test-mutation-ci: fetch-mutation ## Run mutation tests as part of auto build process.
```

Comment:
For future reference, if possible please try to keep the same location in the code - it makes the review process a lot easier